### PR TITLE
Adding ability to get nextFrame of an animation.

### PR DESCRIPTION
**Pull request by pekayatt**
As in discussion http://www.cocos2d-x.org/boards/20/topics/29701 pull request aiming to make possible to get in which frame is the animation.

### CODE REVIEW

File: cocos2dx/actions/CCActionInterval.h [hunk 1]
```diff
@@ -822,6 +822,8 @@ class CC_DLL CCAnimate : public CCActionInterval
 
     /** initializes the action with an Animation and will restore the original frame when the animation is over */
     bool initWithAnimation(CCAnimation *pAnimation);
+	
+	int getNextFrame();
```
Comment (minggo) [inline cocos2dx/actions/CCActionInterval.h]: I think it is better to declare like 

int getNextFrame() const;

Comment (pekayatt) [inline cocos2dx/actions/CCActionInterval.h]: Agree. I have done the changes and re-submitted a PR.

## 

Pedro Monteiro Kayatt ( +55 11 968285744)
Co-founder of Naked Monkey (http://nakedmonkeygames.net)
skype: pekayatt
MSN: pekayatt@hotmail.com

> Bachelor in Computer Engineer by Escola Politécnica da USP
> Master of Science in Computer Engineer by Politecnico di Milano

**Comment by pekayatt** [comment]
Have these changes being accepted? Ty!

**Comment by cocosQA** [comment]
Dear pekayatt:
  We have created a new branch 'v3' to replace branch 'develop', and this PR has been moved from 'develop' branch to 'v3' branch, the new PR is https://github.com/cocos2d/cocos2d-x/pull/6494
The old 'develop' branch will be deleted soon, and this PR will be closed accordingly.

**Pull request closed without merging**